libs/designer-ui/src/lib/flyout/index.tsx

```diff
@@ -25,7 +26,7 @@ const defaultTooltipHostStyles: ITooltipHostStyles = {
 };
 
 export const Flyout = React.forwardRef<{ collapseFlyout(): void }, FlyoutProps>(
-  ({ ariaLabel, iconStyle, style, tabIndex = 0, text, tooltipHostStyles, onClick, title }, ref) => {
+  ({ ariaLabel, iconStyle, style, tabIndex = 0, text, tooltipHostStyles, onClick, title, iconSize = 'lg' }, ref) => {
```

Comment:
was lg teh default size?

Reply:
The size that was being used was 28px, I wasn't sure what else to label this as we want something significantly smaller.